armi/utils/__init__.py

```diff
@@ -810,8 +810,14 @@ def safeCopy(src: str, dst: str) -> None:
     if os.path.isdir(dst):
         dst = os.path.join(dst, os.path.basename(src))
     srcSize = os.path.getsize(src)
-    shutil.copyfile(src, dst)
-    shutil.copymode(src, dst)
+    # Historical note: this used to run shutil.copyfile then shutil.copymode,
+    # but the latter clashed with permissions on a linux system. Both copy and
+    # cp -a accomplish the same thing.
```

Comment:
Personal preference. I don't like historical notes in the code. That's what commits and PRs are for.

When I'm looking at the code, I am trying to understand what it is doing, and I would rather have a comment that describes THAT.

Reply:
Will fix!

Reply:
done